pkg/users/account-logs-panel.jsx

```diff
@@ -31,7 +31,7 @@ const _ = cockpit.gettext;
 export function AccountLogs({ name }) {
     const [logins, setLogins] = useState([]);
     useInit(() => {
-        cockpit.spawn(["last", "--time-format", "iso", "-n", 25, name], { environ: ["LC_ALL=C"] })
+        cockpit.spawn(["last", "--time-format", "iso", "-n25", "--fullnames", name], { environ: ["LC_ALL=C"] })
```

Comment:
Was confused for a bit, why isn't this `-w`, but that's the same thing :) I guess for historical raisins.

Reply:
I prefer long options in code for better readability. But `-w` is the same thing indeed.